src/main/java/org/mule/service/http/impl/service/client/GrizzlyHttpClient.java

```diff
@@ -113,11 +116,7 @@
 
   public static final String CUSTOM_MAX_HTTP_PACKET_HEADER_SIZE = SYSTEM_PROPERTY_PREFIX + "http.client.headerSectionSize";
 
-  // By default, we will delegate the redirect to Grizzly. If the system property is set to true Mule will handle redirects
-  // requests.
-  // Currently, Grizzly will fail to handle concurrent redirects if streaming is not enabled.
-  private static final String ENABLE_MULE_REDIRECT_PROPERTY_NAME = SYSTEM_PROPERTY_PREFIX + "http.enableMuleRedirect";
-  private static final boolean enableMuleRedirect = parseBoolean(getProperty(ENABLE_MULE_REDIRECT_PROPERTY_NAME, "false"));
+  private static final boolean enableMuleRedirect = parseBoolean(getProperty(ENABLE_MULE_REDIRECT_PROPERTY, "true"));
```

Comment:
Has this change of default value been made intentionally?

Reply:
It will be enabled by default for master and 4.4.0